presto-native-execution/presto_cpp/main/operators/BroadcastFactory.cpp

```diff
@@ -137,9 +140,25 @@ BroadcastFileReader::BroadcastFileReader(
     velox::memory::MemoryPool* pool)
     : broadcastFileInfo_(std::move(broadcastFileInfo)),
       fileSystem_(fileSystem),
+      asyncDataCache_(velox::cache::AsyncDataCache::getInstance()),
```

Comment:
AsyncDataCache::getInstance may return null. Let's add a check for that.

Reply:
+1